man/man8/sslsniff.8

```diff
@@ -0,0 +1,48 @@
+.TH sslsniff 8  "2016-08-16" "USER COMMANDS"
+.SH NAME
+sslsniff \- Print data passed to OpenSSL. Uses Linux eBPF/bcc.
+.SH SYNOPSIS
+.B sslsniff
+.SH DESCRIPTION
+sslsniff prints data sent to SSL_write and SSL_read OpenSSL functions, allowing
+us to read plain text content before encryption (when writing) and after
+decryption (when reading).
+
+This works reading the second parameter of both functions (*buf).
+
+Since this uses BPF, only the root user can use this tool.
+.SH REQUIREMENTS
+CONFIG_BPF and bcc.
+.SH EXAMPLES
+.TP
+Print all calls to SSL_write and SSL_read system-wide:
+#
+.B sslsniff
+.SH FIELDS
+.TP
+FUNC
+Which function is being called (SSL_write or SSL_read)
+.TP
+TIME
+Time of the command (HH:MM:SS).
```

Comment:
nit: the timestamp is in seconds, not HH:MM:SS (one day we should make timestamps in all the tools consistent!)